### PR TITLE
Dashboardsv2

### PR DESCRIPTION
**Pull request by mrnetops**
* Add Maintainers & Contributors to the README
* Updated screenshot
* Grafana anonymous access, no login screen
* Fastly Dashboard updates
  * Make rate interval a variable
  * 30s auto-refresh
  * Variable labels

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,3 +86,11 @@ FASTLY_API_TOKEN=${FASTLY_API_TOKEN} docker-compose up
 Access the [Grafana][grafana] dashboard via http://localhost:3000.
 
 ![Fastly Dashboard in Grafana](https://raw.githubusercontent.com/peterbourgon/fastly-exporter/master/compose/Fastly-Dashboard.png)
+
+## Credits and contributions
```
Comment (peterbourgon) [inline README.md]: I'm not trying to diminish anyone's work, but contributors and maintainers are trivially identifiable through the GitHub web UI, and that doesn't require ongoing upkeep. Is this necessary?

Comment (mrnetops) [reply README.md]: I will totally admit this is entirely personal and subjective. ;)

My thoughts are as follows.

Given that
* This is a personal github project, with perhaps an implication that the project was entirely by the owner 
* At least one response of "Cool, let's see what Peter has been up to" when I went to demo the full docker-compose build.

I was looking to see if there was a way to highlight that over time this has evolved from a personal to collaborative project and credit the associated folks.

I spot checked projects for inspiration, but I'm admittedly unclear and totally open to suggestion on the best practices for `Credits and contributions`  including just relying on `graphs/contributors` if that's currently the state of the art.

Comment (peterbourgon) [reply README.md]: All else equal, I want to keep projects like this one as low-effort as possible to maintain. If you feel strongly enough to push for this change even after my preference, I won't object again.

Comment (mrnetops) [reply README.md]: Removing for now while I re-contemplate ;)

**Pull request merged**